### ISSUE TITLE
angular-compilers does not report module import errors 

### ISSUE THREAD

**Issue by pbureau** [Urigo/angular-meteor#1884]
Using the following set of compiler packages:

> angular-compilers@0.2.8_1
> angular-html-compiler@0.2.9
> angular-scss-compiler@0.2.9
> angular-typescript-compiler@0.2.9

The compiler doesn't report any error or warning for missing modules, it is a huge pain to debug. Could this be a configuration problem in my project, or is this a design choice?

**Comment by impcyber** [comment]
What king of errors it doesn't report concretely?
If you write `import { Somthing } from 'package'` does it warn you for missing module?

**Comment by pbureau** [comment]
Hi, actually it seems the compiler is not reporting any kind of errors, I can write:
`import {anything} from 'anything';`
and it will not print any warning, but also any other kind of typescript errors are also silent. I have been trying to find an option that silence the compiler, but I cannot find it. My tsconfig.json:

> {
>   "compilerOptions": {
>     "allowSyntheticDefaultImports": true,
>     "baseUrl": ".",
>     "declaration": false,
>     "emitDecoratorMetadata": true,
>     "traceResolution": true,
>     "importHelpers": true,
>     "noEmitHelpers": true,
>     "experimentalDecorators": true,
>     "lib": [
>       "dom",
>       "es2015"
>     ],
>     "module": "commonjs",
>     "moduleResolution": "node",
>     "sourceMap": true,
>     "target": "es2015",
>     "skipLibCheck": true,
>     "stripInternal": true,
>     "noImplicitAny": true
>   },
>   "exclude": [
>     "node_modules",
>     ".meteor",
>     "tools"
>   ],
>   "types": [
>     "typings.d",
>     "types/node",
>     "types/meteor", 
>     "types/hammerjs"
>   ],
>   "compileOnSave": false,
>   "atom": {
>     "rewriteTsconfig": false
>   },
>   "angularCompilerOptions": {
>     "genDir": "aot",
>     "skipMetadataEmit" : true
>   }
> }
>

**Comment by impcyber** [comment]
Hmm...
Try to change `noImplicitAny` to `false`
And could you show `typings.d.ts`?

**Comment by pbureau** [comment]
As far as I can see, changing `noImplicitAny` has no impact on the compiler output, here the typings file, I have to admit I am not sure how it has been generated:
```
/// <reference types="zone.js" />
/// <reference types="@types/meteor" />
/// <reference types="@types/underscore" />
/// <reference types="@types/chai" />
/// <reference types="@types/mocha" />
/// <reference types="@types/node" />

declare module "*.html" {
  const template: string;
  export default template;
}

declare module "*.scss" {
  const style: string;
  export default style;
}

declare module "*.less" {
  const style: string;
  export default style;
}

declare module "*.css" {
  const style: string;
  export default style;
}

declare module "*.sass" {
  const style: string;
  export default style;
}

declare module "meteor/hwillson:stub-collections" {
  import { Mongo } from "meteor/mongo";

  interface IStubCollections {
    stub(collection: Mongo.Collection);
    restore();
  }

  const StubCollections: IStubCollections;

  export default StubCollections;
}

declare module "chai-spies" {
  const chaiSpies: (chai: any, utils: any) => void;

  export = chaiSpies;
}

interface SpyCalledWith extends Chai.Assertion {
  (...args: any[]): void;
  exactly(...args: any[]): void;
}

interface SpyCalledAlways extends Chai.Assertion {
  with: SpyCalledWith;
}

interface SpyCalledAt {
  most(n: number): void;
  least(n: number): void;
}

interface SpyCalled {
  (n?: number): void;
  /**
   * Assert that a spy has been called exactly once
   *
   * @api public
   */
  once: any;
  /**
   * Assert that a spy has been called exactly twice.
   *
   * @api public
   */
  twice: any;
  /**
   * Assert that a spy has been called exactly `n` times.
   *
   * @param {Number} n times
   * @api public
   */
  exactly(n: number): void;
  with: SpyCalledWith;
  /**
   * Assert that a spy has been called `n` or more times.
   *
   * @param {Number} n times
   * @api public
   */
  min(n: number): void;
  /**
   * Assert that a spy has been called `n` or fewer times.
   *
   * @param {Number} n times
   * @api public
   */
  max(n: number): void;
  at: SpyCalledAt;
  above(n: number): void;
  /**
   * Assert that a spy has been called more than `n` times.
   *
   * @param {Number} n times
   * @api public
   */
  gt(n: number): void;
  below(n: number): void;
  /**
   * Assert that a spy has been called less than `n` times.
   *
   * @param {Number} n times
   * @api public
   */
  lt(n: number): void;
}

declare namespace Chai {
  interface ChaiStatic {
    spy(): any;
  }

  interface Assertion {
    called: SpyCalled;
    always: SpyCalledAlways;
  }
}

```

**Comment by impcyber** [comment]
If you are using `angular-compilers` you don't need this declaration of static files like:
declare module "*.html"
declare module "*.css"
and so on...

Sometimes there are bugs with npm pakcages, and i need to run `rm -rf ./node_modules` and `npm install`. It helps... sometimes

**Comment by pbureau** [comment]
For reference, I think this situation is related with `angular-compilers@0.2.8_1`

When I use version `0.2.9_x`, I have errors, but actually they are errors I cannot solve like the missing modules pb reported here: #1882 
And worse, when I try AOT and ROLLUP I get `UNRESOLVED_IMPORT` that don't make sense...

**Comment by impcyber** [comment]
For now i stopped using angular-compilers and switched to angular-cli with meteor-client-bundler, because I decided to separate the client part from the server part, since client part should now be portable for me and should not be directly dependend.

**Comment by ardatan** [comment]
Closed by PR #1889

**Comment by Urigo** [comment]
today we've released new versions for `angular-compilers` and `meteor-client-bundler` as well with a few fixes and to support the latest Meteor 1.6.1, thanks to the great work of @ardatan !
Let us now if everything works as expected.
